builders/integrations/oracles/chainlink.md

```diff
@@ -476,6 +476,6 @@ contract Client is ChainlinkClient {
 !!! note
     The above example uses the pre-deployed LINK token contract address. You also have the option of deploying your own LINK token contract and using that instead.
 
-Once you've deployed the contract on Remix, you can begin to request the volume data. After you make a request, you can check the status of the job by going to the [**Jobs** section of your node](http://localhost:6688/jobs){target=\_blank}.
+Once you've deployed the contract on Remix, you can begin to request the volume data. After you make a request, After you make a request, you can check the job status under the `Jobs section` on your node at `http://localhost:6688/jobs`.
```

Comment:
```suggestion
Once you've deployed the contract on Remix, you can begin to request the volume data. After you make a request, After you make a request, you can check the job status under the **Jobs** section on your node at `http://localhost:6688/jobs`.
```

If it's not code, it shouldn't go in back ticks. If it's a UI element, they should be in bold